mep/people/templates/people/snippets/member_result.html

```diff
@@ -16,9 +16,9 @@
         <div class="birth-death">
             <dt>birth / death</dt>
```

Comment:
IIRC, Josh also suggested the label should be **birth – death** for consistency, rather than using a slash.